learn/configuration/instance_options.md

```diff
@@ -32,20 +32,114 @@ The above flag disables analytics for the Meilisearch instance and does not acce
 
 To configure a Meilisearch instance using environment variables, set the environment variable prior to launching the instance. If you are unsure how to do this, read more about [setting and listing environment variables](https://linuxize.com/post/how-to-set-and-list-environment-variables-in-linux/), or [use a command-line option](#command-line-options-and-flags) instead.
 
-```bash
+:::: tabs
+::: tab UNIX
+
+```sh
 export MEILI_DB_PATH=./meilifiles
 export MEILI_HTTP_ADDR=127.0.0.1:7700
 ./meilisearch
 ```
 
+:::
+
+::: tab Windows
+
+```sh
+set MEILI_DB_PATH=./meilifiles
+set MEILI_HTTP_ADDR=127.0.0.1:7700
+./meilisearch
+```
+
+:::
+::::
+
 In the previous example, `./meilisearch` is the command that launches a Meilisearch instance, while `MEILI_DB_PATH` and `MEILI_HTTP_ADDR` are environment variables that modify this instance's behavior.
 
 Environment variables for command-line flags accept `n`, `no`, `f`, `false`, `off`, and `0` as `false`. An absent environment variable will also be considered as `false`. Any other value is considered `true`.
 
 Environment variables are always identical to the corresponding command-line option, but prepended with `MEILI_` and written in all uppercase. **Some options (e.g., `--import-snapshots`) are not available as environment variables.**
 
+## Configuration file
+
+Meilisearch accepts a configuration file in the `.toml` format as an alternative to command-line options and environment variables. Configuration files can be easily shared and versioned, and allow you to define multiple options.
+
+**When used simultaneously, environment variables overwrite the configuration file, and command-line options overwrite environment variables.**
+
+You can download the configuration file using the following command:
+
+```sh
+curl https://raw.githubusercontent.com/meilisearch/meilisearch/main/config.toml > config.toml
+```
+
+By default, Meilisearch will use `config.toml` in the working directory as the default configuration file. You will see it when you launch Meilisearch:
```

Comment:
What if you launch Meilisearch w/o a configuration file, does the `Config file path` field still appear in the startup message? If so, what is its value?

```suggestion
By default, Meilisearch will look for a `config.toml` file in the working directory. If it is present, it will be used as the configuration file. You can verify this when you launch Meilisearch:
```